src/main/java/fish/payara/extras/upgrade/BaseUpgradeCommand.java

```diff
@@ -225,11 +225,9 @@ protected boolean reinstallSSHNode(Node node) {
         LOGGER.log(Level.INFO, "Reinstalling SSH node {0}", new Object[]{node.getName()});
         ArrayList<String> command = new ArrayList<>();
         command.add(SystemPropertyConstants.getAdminScriptLocation(glassfishDir));
+        command.add("install-node-ssh");
         command.add("--interactive=false");
-        command.add("--passwordfile");
```

Comment:
@arieki, just to confirm, removing the `--passwordfile` argument should make the `install-node-ssh` command work when a key file is used? Otherwise I don't see immediately how this commit fixes FISH-6240

Reply:
I have tested this using keyfile authentication, without this change, I get the error mentioned in JIRA, with these changes, this works as expected when using keyfile authentication. Extra logging information is also produced which is helpful.

Reply:
Doesn't this break it for passwords authentication?

Reply:
Password authentication still works.